.node-version

```diff
@@ -0,0 +1 @@
+8.9.4
```

Comment:
Dunno if anybody else uses `nodenv`, but this seems like something we should be consistent about and/or document.